nautobot/dcim/models/devices.py

```diff
@@ -1425,6 +1431,9 @@ def clean(self):
                     {"location": f'Devices may not associate to locations of type "{self.location.location_type}".'}
                 )
 
+    def get_capabilities_display(self):
+        return ", ".join(f"{v}" for v in self.capabilities)
```

Comment:
Shouldn't this just be
```suggestion
        return ", ".join(self.capabilities)
```